lib/ansible/modules/network/cumulus/_cl_ports.py

```diff
@@ -19,7 +19,10 @@
 version_added: "2.1"
 author: "Cumulus Networks (@CumulusNetworks)"
 short_description: Configure Cumulus Switch port attributes (ports.conf)
-deprecated: Deprecated in 2.3. Use M(nclu) instead.
+deprecated:
+  removed_in: "2.3"
```

Comment:
I still see code in this file. Should it be removed_in: "2.7" ?

Reply:
Reviewing these by hand again